### ISSUE TITLE
ng-message over multiple lines does not show correctly

### ISSUE THREAD

**Issue by frodefi** [angular/material#3766]
From the docs, I altered [this codepen](http://codepen.io/anon/pen/PqBwdz) to have a longer error message for the name field. If your browser window is not so wide, then the error message goes over two lines, but it does not show correctly as the text is mixed up with the text for the field below, see image (tested on Chrome, Firefox and Safari):
![screen shot 2015-07-16 at 16 52 54](https://cloud.githubusercontent.com/assets/993333/8725246/dc415968-2bdb-11e5-84b8-0566c07b8cb4.png)

**Comment by ivanzvakoms** [comment]
yeah, this issue is because position: absoulte in

```
md-input-container ng-message, md-input-container data-ng-message, md-input-container x-ng-message, md-input-container [ng-message], md-input-container [data-ng-message], md-input-container [x-ng-message], md-input-container .md-char-counter {
    -webkit-font-smoothing: antialiased;
    position: absolute;
    font-size: 12px;
    line-height: 24px; }
```

**Comment by gustavohenke** [comment]
Duplicate of #1952

**Comment by topherfangio** [comment]
This should be fixed in the 1.0.0 release (actually earlier) as we now calculate the height of the messages. Please post a new Codepen if you are still experiencing the issue and we will reopen.

**Comment by ivanzvakoms** [comment]
I've checked. Indeed. This issue is fixed in 1.0.0.
